Makefile

```diff
@@ -54,7 +54,7 @@ test-e2e:
 test-e2e-polybft:
     # We can not build with race because of a bug in boltdb dependency
 	go build -o artifacts/polygon-edge .
-	env EDGE_BINARY=${PWD}/artifacts/polygon-edge E2E_TESTS=true E2E_LOGS=true go test -v -timeout=30m ./e2e-polybft/...
+	env EDGE_BINARY=${PWD}/artifacts/polygon-edge E2E_TESTS=true E2E_LOGS=true go test -race -v -timeout=30m ./e2e-polybft/...
```

Comment:
This target relates to polybft e2e tests.

Unit tests are executed using `test` target, so maybe we should consider adding race flag there (although not sure do we want to open that Pandora box, since it executes all the unit tests both from Edge and Polybft ones)?

Reply:
Indeed, but this can't run with flag because it reports some issues that are not polybft. 
However, polybft e2e tests are testing specific target and that is why I added it. Do you think we should skip race flag until boltdb issue is fixed?